porting/introduction/Preparations.rst

```diff
@@ -3,43 +3,59 @@
 Preparations
 ============
 
-Not all devices can be made to run Ubuntu Touch. Research your target device before you start.
+Not all devices can be made to run Ubuntu Touch.
+Research your specific target device before starting.
 
-To determine your device's specifications, search for it on `GSM Arena <https://www.gsmarena.com>`_ and/or `Device Specification <https://www.devicespecifications.com/en>`_.
-Refer to `Pick an Android target device in the Halium Porting Guide <http://docs.halium.org/en/latest/porting/first-steps.html#pick-an-android-target-device>`_ for further information on requirements and how to check if your device qualifies.
+Find the specifications of your device by searching for it on `GSMArena <https://www.gsmarena.com>`_ and/or `DeviceSpecification <https://www.devicespecifications.com/en>`_.
+`Pick an Android target device<http://docs.halium.org/en/latest/porting/first-steps.html#pick-an-android-target-device>`_ in the Halium Porting Guide helps you find out if it meets the requirements.
```

Comment:
It fails to build here https://ci.ubports.com/blue/organizations/jenkins/docs.ubports.com/detail/PR-524/4/pipeline
Not sure why though :( maybe the deleted space after "device"?